mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -122,9 +122,13 @@ void CGameMode_KZ::SetGameModeVars()
 {
     CGameModeBase::SetGameModeVars();
 
-    // KZ-specific
+    // KZ-specific (Based on SimpleKZ variables)
+    sv_accelerate.SetValue("6.5");
```

Comment:
I know this just a draft, but you can use a float literal here instead of a string
```suggestion
    sv_accelerate.SetValue(6.5f);
```